app/middlewareProtected/page.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function page() {
```

Comment:
Can we maybe give this example page a PascalCase name please? I know it's only an example component but let's keep our patterns :)

Reply:
Done! I actually thought this would throw a build error :thinking: